AFQ/api.py

```diff
@@ -92,7 +92,7 @@ class BundleDict(MutableMapping):
     def __init__(self,
                  bundle_info=BUNDLES,
                  seg_algo="afq",
-                 resample_to=False):
+                 resample_to=afd.read_mni_template()):
```

Comment:
This worries me a little bit. Having a function call in the signature of another function seems a bit risky. Doesn't this trigger the mni download on import? How about setting this to `None` and then have the default behavior be as follows? 

```
if resample_to is None: 
    resample_to = afd.read_mni_template()
```

Reply:
Or is that too confusing? Because `None` sounds like you are not resampling?